sql/mysqld.cc

```diff
@@ -1238,6 +1238,9 @@ bool opt_libcoredumper, opt_corefile = 0;
 bool opt_persist_sensitive_variables_in_plaintext{true};
 int argc_cached;
 char **argv_cached;
+#ifdef HAVE_PERCONA_TELEMETRY
+bool opt_percona_telemetry_disable = false;
```

Comment:
Just to recheck. As I understand it, telemetry is disabled by default. Should this default to true (just in case)?

Reply:
Yes, it should be disabled and not built-in by default. The reason for this is that it makes sense to have telemetry component only with Telemetry Agent package being dependency to PS package. And this is only the case for packages created by Percona. So we will build-in telemetry in our packages, while if someone clones the repo and wants to build PS by himself, there is no point in having telemetry component there.